test/unit/theory/theory_int_opt_white.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+#include "api/cvc4cpp.h"
+#include "smt/optimization_solver.h"
+#include "expr/expr_manager.h"
```

Comment:
Remove

Reply:
Hi Aina, Thanks for the quick review. Just to check, would it be fine to leave in the smt/optimization_solver.h include since I use the class in the test or should I remove it / put the include somewhere else? Thanks again.

Reply:
Only remove line 4 (including the expr_manager).

Reply:
And as @yoni206 pointed out, also line 2.